src/Symfony/Component/Process/Tests/ProcessFailedExceptionTest.php

```diff
@@ -80,11 +81,15 @@ public function testProcessFailedExceptionPopulatesInformationFromProcessOutput(
         $process->expects($this->once())
             ->method('isOutputDisabled')
             ->will($this->returnValue(false));
+			
+        $process->expects($this->once())
+            ->method('getWorkingDirectory')
+            ->will($this->returnValue($workingDirectory));
 
         $exception = new ProcessFailedException($process);
 
         $this->assertEquals(
-            "The command \"$cmd\" failed.\nExit Code: $exitCode($exitText)\n\nOutput:\n================\n{$output}\n\nError Output:\n================\n{$errorOutput}",
+            "The command \"$cmd\" failed.\n\nExit Code: $exitCode($exitText)\n\nWorking directory: {$workingDirectory}\n\nOutput:\n================\n{$output}\n\nError Output:\n================\n{$errorOutput}",
```

Comment:
[The rest of the exception message](https://github.com/Rvanlaak/symfony/blob/patch-2/src/Symfony/Component/Process/Exception/ProcessFailedException.php#L39) also uses two line breaks, so while I was fixing the tests I've made it consistent by also adding them over here.